components/brave_rewards/browser/test/rewards_browsertest.cc

```diff
@@ -14,6 +14,7 @@
 #include "base/strings/string_split.h"
 #include "base/strings/stringprintf.h"
 #include "base/test/bind_test_util.h"
+#include "base/test/mock_callback.h"
```

Comment:
is this needed?

Reply:
no, leftover from trying to mock calls differently